PatientProj/src/hospital/Health_Info.java

```diff
@@ -1,7 +1,71 @@
 package hospital;
 
-import java.util.Scanner;
+import java.util.*;
```

Comment:
Static imports with (*) matchers should be avoided. to reduce memory build up during compile time.
http://stackoverflow.com/questions/187453/import-package-vs-import-package-specifictype